Decred Wallet/Features/Transactions/TransactionTableViewCell.swift

```diff
@@ -0,0 +1,104 @@
+        }
+
+        self.txDateLabel.text = isConfirmed ? txDateString : LocalizedStrings.pending
+        self.txDateLabel.textColor = isConfirmed ? UIColor.appColors.bluishGray : UIColor.appColors.lightBluishGray
+        self.txStatusIconImageView.image = isConfirmed ? UIImage(named: "ic_confirmed") : UIImage(named: "ic_pending")
+
+        self.stakingTxAmountLabel.isHidden = transaction.type == DcrlibwalletTxTypeRegular
+        self.daysCounterLabel.isHidden = transaction.type == DcrlibwalletTxTypeRegular
+        self.voteRewardLabel.isHidden = true //TODO: set value
+
+        if transaction.type == DcrlibwalletTxTypeRegular {
+            self.displayRegularTxInfo(transaction)
+        } else if transaction.type == DcrlibwalletTxTypeVote {
+            self.displayVoteTxInfo(transaction)
+        } else if transaction.type == DcrlibwalletTxTypeTicketPurchase {
+            self.displayTicketPurchaseInfo(transaction)
+        }
+    }
+    
+    func displayRegularTxInfo(_ transaction: Transaction) {
+        let amountString = Utils.getAttributedString(str: transaction.dcrAmount.round(8).description, siz: 13.0, TexthexColor: UIColor.appColors.darkBlue)
+
+        if transaction.direction == DcrlibwalletTxDirectionSent {
+            let attributedString = NSMutableAttributedString(string:"-")
+            attributedString.append(amountString)
+            self.txAmountOrTicketStatusLabel.attributedText = attributedString
+            self.txTypeIconImageView?.image = UIImage(named: "ic_send")
+        } else if transaction.direction == DcrlibwalletTxDirectionReceived {
+            self.txAmountOrTicketStatusLabel.attributedText = amountString
+            self.txTypeIconImageView?.image = UIImage(named: "ic_receive")
+        } else if transaction.direction == DcrlibwalletTxDirectionTransferred {
+            self.txAmountOrTicketStatusLabel.attributedText = amountString
+            self.txTypeIconImageView?.image = UIImage(named: "ic_fee")
+        }
+    }
+    
+    func displayTicketPurchaseInfo(_ transaction: Transaction) {
+        self.txAmountOrTicketStatusLabel.text = "\(LocalizedStrings.voted)"
+        self.txTypeIconImageView?.image = UIImage(named: "ic_ticketVoted")
+
+        self.stakingTxAmountLabel.attributedText = Utils.getAttributedString(str: transaction.dcrAmount.round(8).description, siz: 11.0, TexthexColor: UIColor.appColors.lightBluishGray)
+        self.daysCounterLabel.text = String(format: LocalizedStrings.days, -Date(timeIntervalSince1970: TimeInterval(transaction.timestamp)).daysFromNow)
+    }
+    
+    func displayVoteTxInfo(_ transaction: Transaction) {
+        self.txAmountOrTicketStatusLabel.text = "\(LocalizedStrings.ticket)"
+        self.txTypeIconImageView?.image = UIImage(named: "ic_ticketImmature")
+        self.daysCounterLabel.text = String(format: LocalizedStrings.days, -Date(timeIntervalSince1970: TimeInterval(transaction.timestamp)).daysFromNow)
```

Comment:
Can pass in the `ageInDays` variable from `displayInfo` function and thus simplify this code.